ros/lib/models.py

```diff
@@ -10,12 +10,16 @@ class PerformanceProfile(db.Model):
     performance_record = db.Column(JSONB)
     performance_utilization = db.Column(JSONB)
     report_date = db.Column(
-        db.DateTime(timezone=True), default=datetime.datetime.utcnow)
+        db.DateTime(timezone=True),
+        default=datetime.datetime.utcnow,
+        nullable=False
+    )
```

Comment:
In my opinion, this change should go separately as it is not dependent on psi-enabled. This change is now blocked on this PR to get approved so to drive such common changes quickly, it would be good to create a separate PR. 
Currently, it's not blocking anyone's development so for now, it is OK.